apps/condo/domains/billing/schema/BillingIntegration.js

```diff
@@ -87,12 +89,11 @@ const BillingIntegration = new GQLListSchema('BillingIntegration', {
             },
         },
 
-        currency: {
+        currencyCode: {
+            ...CURRENCY_CODE_FIELD,
             schemaDoc: 'Currency which this billing uses',
-            type: Relationship,
-            ref: 'BillingCurrency',
-            isRequired: false,
-            kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
+            isRequired: true,
+            defaultValue: DEFAULT_CURRENCY_CODE,
```

Comment:
I think we should force integration to specify currencyCode values, because default value can be used to create mistakes. Clarifying it will decrease amount of errors

Reply:
> We already force integration to specify (isRequired: true)

Are we? I thought it's working just like default value in DB, so it will be passed, if not specified....It means if I forgot to specify it somewhere, defaultValue will be passed....But no error will be thrown, it can lead to mistakes later

Reply:
Oh, let me double check it